docs/sphinx/methods/embedding.rst

```diff
@@ -0,0 +1,107 @@
+.. _`sec:methods:embedding`:
+
+Active Space Embedding Theory
+=======================================
+
+.. codeauthor:: Nan He
```

Comment:
Maybe CiCi should be acknowledged for the AVAS part.

Reply:
I agree. Can you add her name please?

Reply:
This document seems to be for embedding, I can also make a section for AVAS in the same docs if needed? Cause they are seperate functions currently.